code/game/machinery/tcomms/tcomms_core.dm

```diff
@@ -27,6 +27,7 @@
 	/// What tab of the UI were currently on
 	var/ui_tab = UI_TAB_CONFIG
 
+
 /**
   * Initializer for the core.
   *
```

Comment:
```suggestion
	/// What tab of the UI were currently on
	var/ui_tab = UI_TAB_CONFIG

/**
  * Initializer for the core.
  *
```
Added by accident i imagine

Reply:
Yep, I was adding it to the tcomms core originally before realizing it was better in the radio. oops!